### ISSUE TITLE
Add groups membership to DSAccount class?

### ISSUE THREAD

**Issue by ciyi** [MichaelGrafnetter/DSInternals#82]
Hey!

Thank you so much for the amazing work on DSInternals. 

I'm interested in knowing if you plan to add Groups membership of users? Ultimately to be listed when using the PowerShell cmdlet Get-ADDBAccount so it'd show all the groups the user/s are member of.

Thank you very much in advance,
Carlos

**Comment by MichaelGrafnetter** [comment]
Hi @ciyi , it is on the TODO list, but with a low priority, as this info can be retrieved from a ntds.dit file using the built-in `dsamain.exe` + `Get-ADPrincipalGroupMembership`.

**Comment by ciyi** [comment]
Thanks for replying Michael. dsamain.exe could be a good temporary solution, it's just that I believe I'd need RSAT to be installed :(

**Comment by MichaelGrafnetter** [comment]
Yes, you need RSAT (and AD LDS I think). But you can install them on Windows 7-10.
